pkg/cli/root.go

```diff
@@ -412,3 +414,28 @@ func LoadConfiguration(config *config.Config, configPath string) error {
 
 	return nil
 }
+
+func isDefaultPolicyConfig(cfg *config.Config) bool {
+	adminPolicy := cfg.AccessControl.AdminPolicy
+
+	log.Info().Msg("checking if default authorization is possible")
+
+	if len(adminPolicy.Actions)+len(adminPolicy.Users) > 0 {
+		log.Info().Msg("default policy config can't have admin policy")
```

Comment:
I think this logs are misleading, same with below.

Maybe add a log.Info().Msg("authz with only default policies detected")

Reply:
`func isDefaultPolicyConfig(cfg *config.Config) bool {
	adminPolicy := cfg.AccessControl.AdminPolicy

	log.Info().Msg("checking if default authorization is possible")

	if len(adminPolicy.Actions)+len(adminPolicy.Users) > 0 {
		log.Info().Msg("admin policy detected, default authorization disabled")

		return false
	}

	for _, repository := range cfg.AccessControl.Repositories {
		for _, policy := range repository.Policies {
			if len(policy.Actions)+len(policy.Users) > 0 {
				log.Info().Interface("repository", repository).
					Msg("repository with non-empty policy detected, default authorization disabled")

				return false
			}
		}
	}

	log.Info().Msg("default authorization detected")

	return true
}`
Are these logs more intuitive?